### PR TITLE
Autopet page

### PR DESCRIPTION
**Pull request by Vladislav94**

### CODE REVIEW

File: src/pages/Autopet/components/AutopetSteps/PetPanel.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            className={classes.tabPanel}
+        >
+            <Typography className={classes.panelText}>
+                Approve contract to pet your gotchi(s)
+            <Typography className={classes.panelText}>
+                This is trustless, contract can only pet your gotchi, nothing else
+            </Typography>
+            </Typography>
+            <Typography className={classes.panelText}>
+                You can take back pet right anytime
```
Comment (dudendy) [inline src/pages/Autopet/components/AutopetSteps/PetPanel.js:31]: typo - `rights` not `right`

File: package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "axios": "^0.21.1",
     "classnames": "^2.2.6",
     "cross-fetch": "^3.1.4",
-    "ethers": "^5.4.6",
+    "ethers": "^5.4.0",
```
Comment (dudendy) [inline package.json:21]: let's use last `ethers` version

File: src/pages/Autopet/components/AutopetInfo/AutopetInfo.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { Link } from "@mui/material";
+import { useContext } from "react";
+import { AutopetContext } from "../../AutopetContextProvider";
+import { infoStyles } from "../../styles";
+import AutopetInfoCard from "./AutopetInfoCard";
+import CallMadeIcon from '@mui/icons-material/CallMade';
+import classNames from "classnames";
```
Comment (dudendy) [inline src/pages/Autopet/components/AutopetInfo/AutopetInfo.js:7]: please change double-quotes to single-quotes (all new files)

File: src/api/common/ethersApi.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        case 'test':
+            return testProvider;
+        default:
+            return polygonProvider;
+    }
+}
+
+// eslint-disable-next-line import/no-anonymous-default-export
+export default {
+    async waitForTransaction(hash, network) {
```
[the remark this answers is not in the document]
Comment (dmutrokot) [reply src/api/common/ethersApi.js:18]: discussed, not needed to separate

**Review by dudendy — APPROVED** [review]

**Pull request merged**